api/src/main/resources/workbench-api.yaml

```diff
@@ -2432,6 +2432,41 @@ paths:
           description: The Terra Job for the extraction workflow
           schema:
             "$ref": "#/definitions/GenomicExtractionJob"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/abortExtract/{wgsCohortExtractionJobId}":
```

Comment:
To make this match the usual RESTful patterns (though this is definitely a custom verb), my general preference is to match REST for the noun, then tack a verb onto the end. In this case:

```
/v1/workspaces/{workspaceNamespace}/{workspaceId}/genomicExtractionJobs/{jobId}
```

Reply:
Sorry I wrote this too fast and omitted the verb:

```
/v1/workspaces/{workspaceNamespace}/{workspaceId}/genomicExtractionJobs/{jobId}/abort
```